src/number/numericString.ts

```diff
@@ -15,18 +17,53 @@ export class NumericString {
       if (match == null) {
         continue;
       }
-      return new NumericString(parseInt(match[0], base), base, prefix);
+
+      // Regex to determine if this number has letters around it,
+      // if it doesn't then that is easy and no prefix or suffix is needed
+      let findNondigits = /[^\d-+]+/i;
+
+      // Regex to find any leading characters before the number
+      let findPrefix = /^[^\d-+]+(?=[0-9]+)/i;
+
+      // Regex to find any trailing characters after the number
+      let findSuffix = /[^\d]*$/i;
+
+      let newPrefix = prefix;
+      let newSuffix = "";
+      let newNum = input;
+
+      // Only use this section if this is a number surrounded by letters
+      if (findNondigits.exec(input) !== null && NumericString.matchings[NumericString.matchings.length - 1].regex === regex) {
```

Comment:
Would like feedback on how I can get an ID to only run this for 1 of the 4 regex situations